indexers/social_feed/social_feed.js

```diff
@@ -166,70 +166,74 @@ async function getBlock(block: Block) {
+                                        account_id: {_eq: $accountId},
+                                        block_height: {_eq: $blockHeight}
+                                    },
+                                    limit: 1
+                                ) {
+                                    account_id
+                                    accounts_liked
+                                    block_height
+                                    block_timestamp
+                                    content
+                                    id
+                                }
+                            }`,
+              {
+                accountId: itemAuthor,
+                blockHeight: itemBlockHeight,
+              },
+            );
+            if (posts.dataplatform_near_social_feed_posts.length == 0) {
+              return;
+            }
 
-          const post = posts.dataplatform_near_social_feed_posts[0];
-          switch (likeAction) {
-            case "like":
-              await _handlePostLike(
-                post.id,
-                accountId,
-                blockHeight,
-                blockTimestamp,
-                receiptId,
-              );
-              break;
-            case "unlike":
-            default:
-              await _handlePostUnlike(post.id, accountId);
-              break;
+            const post = posts.dataplatform_near_social_feed_posts[0];
+            switch (likeAction) {
+              case "like":
+                await _handlePostLike(
+                  post.id,
+                  accountId,
+                  blockHeight,
+                  blockTimestamp,
+                  receiptId,
+                );
+                break;
+              case "unlike":
+              default:
+                await _handlePostUnlike(post.id, accountId);
+                break;
+            }
+          } catch (e) {
+            console.log(
+              `Failed to store like to post ${itemAuthor}/${itemBlockHeight} as we don't have it stored in the first place.`,
+            );
           }
-        } catch (e) {
-          console.log(
-            `Failed to store like to post ${itemAuthor}/${itemBlockHeight} as we don't have it stored in the first place.`,
-          );
-        }
-        break;
-      case "comment":
-        // Comment
-        console.log(`Likes to comments are not supported yet. Skipping`);
-        break;
-      default:
-        // something else
-        console.log(`Got unsupported like type "${itemType}". Skipping...`);
-        break;
+          break;
+        case "comment":
+          // Comment
+          console.log(`Likes to comments are not supported yet. Skipping`);
+          break;
+        default:
+          // something else
+          console.log(`Got unsupported like type "${itemType}". Skipping...`);
+          break;
+      }
+    } catch (likeError) {
+      console.log("Error in handleLike", likeError);
```

Comment:
Can we instead say something like "Failed to process like acction content."? Since the rest of the code is in try catches, we know that this one should catch an error related to reading the json and preparing variables.

Reply:
I'll just close this in favor of your PR.